snowpack/package.json

```diff
@@ -50,16 +50,18 @@
   "dependencies": {
     "@snowpack/plugin-build-script": "^2.0.11",
     "@snowpack/plugin-run-script": "^2.2.0",
+    "@types/cheerio": "^0.22.22",
     "cacache": "^15.0.0",
     "cachedir": "^2.3.0",
+    "cheerio": "^1.0.0-rc.3",
```

Comment:
Loose suggestion: if you‘re only using `cheerio` to find specific elements like `<script>` or `<link>`, [hypertag](https://github.com/AndreasPizsa/hypertag) works well for this (and even parses attributes). It could be a faster & much lighter-weight alternative. But if you need cheerio for something heavier in the future, it‘s fine

Reply:
yea, the only reason I switched it out for cheerio was because we do a lot more with it. You can see in the file we're querying, modifying, adding and removing different elements as we optimize.